### ISSUE TITLE
"Generate a Trello key and token" link is broken, can't set up

### ISSUE THREAD

**Issue by rootwork** [patbenatar/trello-search-workflow#1]
Great looking workflow!

Unfortunately the link in the docs to "Generate a Trello key and token" is broken -- https://trello.com/docs/gettingstarted/#getting-an-application-key just redirects to https://developers.trello.com/

I was able to search around and get a key, but to get a token it looks like there are several options, and I wasn't able to figure out how to directly get one with any of them.

So at this point I haven't been able to get this set up at all.

**Comment by patbenatar** [comment]
@rootwork uh oh, sounds like I need to update the readme! Thanks for pointing this out.

**Comment by patbenatar** [comment]
@rootwork Can you try these instructions?
1. Download and install [Trello Board Search.alfredworkflow](https://github.com/patbenatar/trello-search-workflow/blob/master/Trello Board Search.alfredworkflow?raw=true)
2. Visit [https://trello.com/app-key](https://trello.com/app-key) and copy your Developer API Key
3. Visit this URL, replacing [YOUR_KEY] with your key from the previous step: https://trello.com/1/authorize?expiration=never&scope=read&response_type=token&name=Trello%20Search%20Workflow&key=[YOUR_KEY]
4. Click Allow
5. Copy the token
6. Put your key from step 2 in `.auth_key` and token from step 5 in `.auth_token` (at root of workflow directory)

**Comment by Jawwads** [comment]
@patbenatar, sorry for a silly question. But can you please help me with point #6. I don't understand what you mean by ".auth_key" or ".auth_token"? Do I create .txt files in the root directory as follows? Appreciate the help as I am learning this. 

![image](https://cloud.githubusercontent.com/assets/2739634/22622462/d2a3b93a-eb08-11e6-9ccf-d0b1fc719daf.png)

**Comment by patbenatar** [comment]
@Jawwads no such thing as a silly question 😄 

You're very close, just rename those files to have a dot in front of them. `.auth_key` and `.auth_token`

**Comment by Jawwads** [comment]
Alright, then hoping you @patbenatar won't mind this silly question either; 

When I try to rename them with a ".", I get the following error
![image](https://cloud.githubusercontent.com/assets/2739634/22655124/22bd2ce4-ec5d-11e6-97be-a6ee54354dfc.png)

**Comment by patbenatar** [comment]
@Jawwads Yeah, Finder tries to prevent you from making dotfiles but you can do it from Terminal. 

In Terminal, find that directory by `cd`ing into it. Then you can rename the files like so: `mv auth_key .auth_key` and `mv auth_token .auth_token`

**Comment by Jawwads** [comment]
Thank you so much! I am going to try it and let you know. 

Appreciate you helping me learn :) 👍
